src/App.tsx

```diff
@@ -76,7 +77,10 @@ const App: FC = () => {
           dispatch(setNotification(feideUser.error, NotificationVariant.Error));
           setIsLoadingUser(false);
         } else {
-          dispatch(setUser(feideUser));
+          const cristinId = await getCustomerInstitution(feideUser['custom:customerId']).then(
+            (customer) => customer?.data?.cristinId
+          );
```

Comment:
jeg trodde vi skulle få dette direkte via innlogging nå, og ikke måtte gjøre ekstra kall for det? (`custom:organizationId`)
https://unit.atlassian.net/browse/NP-1272?atlOrigin=eyJpIjoiYzExODRiMmFlMDUyNGNjMGFmNTEyZjdlMWM5Njc3MWQiLCJwIjoiaiJ9

Reply:
åja! men det er ikke det vi får nå hvertfall - nå får vi organizationId=organizationNumber, får ikke cristinId 😛  vet du hva som skjedde med cristinId, @janbj ?